lib/event-highlighter.js

```diff
@@ -0,0 +1,318 @@
+'use babel'
+
+import { Range, Point, CompositeDisposable } from "atom"; // eslint-disable-line no-unused-vars
+import path from "path"; // eslint-disable-line no-unused-vars
+const osc = require('osc-min');
```

Comment:
these lines can be deleted

Reply:
Done - I really have poor dx on my private machine. This is most likely, because I use Pulsar for editing the Pulsar package :-D Maybe I will think about optimizing this.